content/v2.0/reference/api/client-libraries/python.md

```diff
@@ -92,7 +94,7 @@ org = "<my-org>"
 token = "<my-token>"
 
 client = influxdb_client.InfluxDBClient(
-    url="http://localhost:9999",
+    url="https://us-west-2-1.aws.cloud2.influxdata.com",
```

Comment:
This should actually stay `http://localhost:9999`, that way it will get replaced by the InfluxDB URL selection.